app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/address/CompanyDetailsController.scala

```diff
@@ -103,8 +134,15 @@ class CompanyDetailsController @Inject() (
     request: Request[_]
   ): EitherT[Future, Error, JourneyStatus] = {
     val newJourney = journey.journey.copy(
-      draftReturn = journey.draftReturn.copy(
-        companyAddress = Some(address)
+      draftReturn = journey.draftReturn.fold(
+        _.copy(exampleCompanyDetailsAnswers =
+          Some(
+            IncompleteExampleCompanyDetailsAnswers.empty.copy(
```

Comment:
here you're unsetting any disposal price or acquisition price the user has entered - is that correct?